illumina.py

```diff
@@ -507,6 +507,19 @@ def fetch_by_index(self, idx):
 # ***  miseq_fastq_to_bam   ***
 # =============================
 
+def sam_to_fastq(outFastq1, outFastq2, inBam, clippingAction=None, clippingAttribute=None,
```

Comment:
You can delete these additions to illumina.py (which may have been better housed at read_utils.py anyway) as they are unnecessary if you make the requested changes to the kraken and krona methods.